tests/performance/insert_select_squashing.xml

```diff
@@ -0,0 +1,23 @@
+<test>
+    <settings>
+        <max_block_size>1000</max_block_size>
+    </settings>
+
+    <create_query>
+CREATE TABLE squash_performance
+(
+    s1 String,
+    s2 Nullable(String),
+    a1 Array(Array(String)),
+    a2 Array(Array(UInt32)),
+    m1 Map(String, Array(String)),
+    m2 Map(String, Array(UInt64)),
+    t Tuple(String, Array(String), Map(String, String))
```

Comment:
would it make sense to test also Variant and Dynamic columns?

Reply:
Sure, I wanted to add separate test for them but just forgot about it. Will add them